lib/modules/dosomething/dosomething_global/dosomething_global.module

```diff
@@ -5,3 +5,22 @@
  */
 
 include_once 'dosomething_global.features.inc';
+
+/**
+ * Implements hook_form_alter().
+ */
+function dosomething_global_form_campaign_node_form_alter(&$form, &$form_state, $form_id) {
+  global $user;
+  // Check if the user is a mexico or brazil admin
+  if (in_array('brazil admin', $user->roles) || in_array('mexico admin', $user->roles)) {
+    // Get translatable fields.
+    $fields = $form_state['field'];
+    foreach ($fields as $field) {
+      $translatable = $field['en']['field']['translatable'];
+      // If the field is not translatable disable access 
```

Comment:
instead of checking the lang @sergii-tkachenko suggested testing with `field_info`
https://github.com/DoSomething/phoenix/blob/dev/lib/modules/dosomething/dosomething_global/dosomething_global.drush.inc#L145-L152